### ISSUE TITLE
Add Selftracking / Center on self

### ISSUE THREAD

**Issue by otting** [WolfCorps/TacControl#7]
It would be nice to have the map follow the active player.

**Comment by dedmen** [comment]
TacControl doesn't know where you are.
Its intended to not be "magic", but I can add the Feature to follow a GPS Tracker.

**Comment by otting** [comment]
Do you plan on adding a personal gps device then? I only found the "explosive" that is used for vehicles.

**Comment by dedmen** [comment]
not currently planned. But would be nice to have defintiely.

**Comment by SekiAlpha** [comment]
Maybe you can use the ace microdagr as a gps tracking item?

**Comment by SekiAlpha** [comment]
If you do add this feature it would be nice to filter out people and have an option to only see group team leads or make it so you can manually group people up and filter through the groups.

**Comment by otting** [comment]
@SekiAlpha TacControl only takes what is on your map, it might be easier to handle the blueforce tracking directly ingame. In our configuration (pretty standard I think) only group leaders are shown on the map and that is exactly what you see on TacControl

**Comment by dedmen** [comment]
Well thats left to your group then.
Only give leaders GPS items then.

**Comment by SekiAlpha** [comment]
> 
> 
> @SekiAlpha TacControl only takes what is on your map, it might be easier to handle the blueforce tracking directly ingame. In our configuration (pretty standard I think) only group leaders are shown on the map and that is exactly what you see on TacControl

Yes, it would be nice if I could do that, however in my unit we use CTAB for BLUEFOR tracking. Either way, I'd like there to be a method to track people using tac control and manually group them up. It would be a powerful feature that most BLUEFOR trackers don't have.
